### PR TITLE
[Method Ref 2] LPS-164448 12-5

### PR DESCRIPTION
**Pull request by JanisZhang**
Resend. Fixed Integration test.

Previous:https://github.com/liferay-core-infra/liferay-portal/pull/2606

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

**Comment by JanisZhang** [comment]
code conflicts.

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-service/src/main/java/com/liferay/portal/reports/engine/console/service/impl/EntryLocalServiceImpl.java [hunk 1]
```diff
@@ -123,7 +123,7 @@ public Entry addEntry(
 		entry.setCreateDate(serviceContext.getCreateDate(date));
 		entry.setModifiedDate(serviceContext.getModifiedDate(date));
 		entry.setDefinitionId(definitionId);
-		entry.setFormat(format);
+		entry.setFormat(reportFormat.toString());
```
Comment (tinatian) [inline modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-service/src/main/java/com/liferay/portal/reports/engine/console/service/impl/EntryLocalServiceImpl.java:126]: @dantewang @JanisZhang This changes what stored into database which needs an upgrade for existed data.

**Comment by tinatian** [comment]
@JanisZhang @dantewang Please see https://github.com/liferay-core-infra/liferay-portal/pull/2769, I only take part of the commits, because other changes actually changed the behavior of the api method, for example, before it passed in lower case values, and after your change, it needs an upper case value. This actually changes the behavior, which is not we want, we should only focus on backend structure without behavior changes.

**Pull request closed without merging**